docs/querying-data.rst

```diff
@@ -0,0 +1,207 @@
+    tender_status;
+
+To learn more about the summaries and aggregates in the ``tender_summary`` table, refer to the :doc:`database`.
+
+.. tip::
+
+  The ``tender``, ``award`` and ``contract`` objects in OCDS all have a ``.status`` property.
+
+  Kingfisher Views exposes these status properties in ``tender_summary.tender_status``, ``awards_summary.award_status`` and ``contracts_summary.contract_status``.
+
+  Consider which statuses you want to include or exclude from your analysis, for example you might wish to exclude pending and cancelled contracts when calculating the total value of contracts for each buyer.
+
+  The `OCDS codelist documentation <https://standard.open-contracting.org/latest/en/schema/codelists/#>`__ describes the meaning of the statuses for each object.
+
+Calculate the top 10 buyers by award value
+------------------------------------------
+
+Summary data on buyers is stored in the ``buyer_summary`` table and summary data on awards is stored in the ``award_summary`` table.
+
+To join summary tables, use the ``id`` column, which uniquely identifies a release. To learn more about the relationships between tables refer to the :doc:`database`.
+
+The ``buyer_summary`` table doesn't include the buyer's name, however the ``buyer`` column contains a jsonb blob of the buyer  for each contracting process, which this can be extracted from.
+
+Most summary tables in Kingfisher Views include a column containing jsonb blobs of the object the summary relates to.
+
+The following query calculates the top 10 buyers by award value for collection ``1060``, disaggregated by currency and counting 'active' awards only:
+
+.. code-block:: sql
+
+  select
+      buyer_identifier,
+      buyer -> 'name' as buyer_name, --extract the buyer name from the JSON
+      award_value_currency,
+      sum(award_value_amount) as award_amount
+  from
+      awards_summary
+  join
+      buyer_summary on awards_summary.id = buyer_summary.id
+  where
+      awards_summary.collection_id = 1060
+  and
+      awards_summary.award_value_amount > 0 --filter out awards with no value
+  and
+      awards_summary.award_status = 'active'
+  group by
+      buyer_identifier,
+      buyer_name,
+      award_value_currency
+  order by
+      award_amount desc
+  limit
+      10;
+
+Querying other collections and fields
+-------------------------------------
+
+Coverage of the OCDS schema varies by publisher.
+
+To identify the fields needed for your analysis and how to answer them, use the `OCDS schema documentation <https://standard.open-contracting.org/latest/en/schema/release/>`__ to understand the meaning, structure and format of the fields in OCDS.
+
+To check whether the fields needed for your analysis are available for a particular collection, use the ``field_counts`` table.
+
+The following query lists the coverage of each field in collection ``1060``:
+
+.. code-block:: sql
+
+  select
+    *
+  from
+    field_counts
+  where
+    collection_id = 1060
```

Comment:
`field_counts` only has results for the selected collections (which is often just one collection). Also, in cases where there are multiple selected collections, the user is presumably interested in coverage across all of them. Should we remove the WHERE clause?

Reply:
Yes, this was originally authored when all the views where in a single schema. 

I've removed it in https://github.com/open-contracting/kingfisher-views/pull/109/commits/ed554bbeca1617577f59d2ee115051cd6c4bb8c1 and added a note for users of schemas with multiple collections (looking at the current list of schemas, it seems fairly common practice for the helpdesk to generate schemas with multiple collections).